src/openms/include/OpenMS/CHEMISTRY/AdductInfo.h

```diff
@@ -0,0 +1,97 @@
+// --------------------------------------------------------------------------
```

Comment:
I made a diff: https://www.diffchecker.com/W9PdsUP4
that shows a few differences. 
Please check as these could be a result of something missed during the merge.
e.g., if a branch already moved code to a new header file and on the other branch changes come in to the old file things might get missed

Reply:
Nothing missed, just some additions.